crates/rpc/rpc/src/debug.rs

```diff
@@ -756,9 +782,58 @@ where
                         return Ok((frame.into(), res.state))
                     }
                     GethDebugBuiltInTracerType::FlatCallTracer => {
-                        return Err(
-                            EthApiError::Unsupported("Flatcall tracer is not supported yet").into()
-                        )
+                        let flat_call_config = tracer_config
+                            .into_flat_call_config()
+                            .map_err(|_| EthApiError::InvalidTracerConfig)?;
+
+                        let mut inspector = TracingInspector::new(
+                            TracingInspectorConfig::from_flat_call_config(&flat_call_config),
+                        );
+
+                        let (res, env) = self.eth_api().inspect(db, env, &mut inspector)?;
+
+                        let tx_info = TransactionInfo {
+                            hash: {
+                                #[cfg(not(feature = "js-tracer"))]
+                                {
+                                    _transaction_context.unwrap().tx_hash
+                                }
+                                #[cfg(feature = "js-tracer")]
+                                {
```

Comment:
lets get rid of this entirely, we can now also remove the feature gated argument because we use it now

also releasing this which makes this conversion a simple into() https://github.com/paradigmxyz/revm-inspectors/pull/204

Reply:
removed `transaction_conext` feature gating in the function. updated conversion to not have feature gate as well. left the feature gating on js-tracer since that seems unrelated